packages/core/src/schema/models/documentLogs.ts

```diff
@@ -34,5 +34,8 @@ export const documentLogs = latitudeSchema.table(
   (table) => ({
     documentLogUuidIdx: index('document_log_uuid_idx').on(table.documentUuid),
     commitIdIdx: index('document_logs_commit_id_idx').on(table.commitId),
+    resolvedContentIdx: index('document_logs_resolved_content_idx').on(
```

Comment:
Wouldn't be too much tralla for an index?

Reply:
yes... I added a new column instead called `contentHash`, which serves the same purpose but with a shorter content.